src/server/templates/typescript.ts

```diff
@@ -178,56 +183,42 @@ export interface Database {
                 : schemaViews.map(
                     (view) => `${JSON.stringify(view.name)}: {
                   Row: {
-                    ${view.columns
-                      .sort(({ name: a }, { name: b }) => a.localeCompare(b))
-                      .map(
-                        (column) =>
-                          `${JSON.stringify(column.name)}: ${pgTypeToTsType(
-                            column.format,
-                            types,
-                            schemas
-                          )} ${column.is_nullable ? '| null' : ''}`
-                      )}
+                    ${columnsByTableId[view.id].map(
```

Comment:
The columns were grouped by table id and you're accessing them via `view.id` here - are you sure you don't need another group by view id for these?